src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -0,0 +1,222 @@
+import { VStack } from "@chakra-ui/react";
+import type { Meta, StoryObj } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import Heading from "../Heading/Heading";
+
+import DatePicker from "./DatePicker";
+
+const currentYear = new Date().getFullYear();
+
+const meta: Meta<typeof DatePicker> = {
+  title: "DatePicker",
+  component: DatePicker,
+  decorators: [withDesign],
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=11689%3A0",
+    },
+    jest: ["DatePicker.test.tsx"],
+  },
+  argTypes: {
+    className: { control: false },
+    dateFormat: {
+      table: { defaultValue: { summary: "yyyy-MM-dd" } },
+    },
+    dateType: {
+      control: { type: "select" },
+      table: { defaultValue: { summary: "full" } },
+    },
+    id: { control: false },
+    isDateRange: {
+      table: { defaultValue: { summary: false } },
+    },
+    isDisabled: {
+      table: { defaultValue: { summary: false } },
+    },
+    isInvalid: {
+      table: { defaultValue: { summary: false } },
+    },
+    isRequired: {
+      table: { defaultValue: { summary: false } },
+    },
+    key: { table: { disable: true } },
+    labelText: {
+      table: { defaultValue: { summary: "From" } },
+    },
+    nameFrom: { control: false },
+    nameTo: { control: false },
+    onChange: { control: false },
```

Comment:
I'm not sure if this is something you want to keep, but I remember the actions I added for MultiSelect ;).
To enable the action log for the onChange function, we would have to add in the argTypes:
`onChange: { control: false, action: "onChange" }`
and in the WithControls story we would omit the onChange prop in the args as well as in the Component (line 92 and 98).

Reply:
Yea, this is nice. I'll add it in for now but we need a defined pattern for actions across all components.